src/components/FloatingDropdown/FloatingDropdown.driver.ts

```diff
@@ -0,0 +1,57 @@
+import { Simulate } from 'react-dom/test-utils';
+
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { UniDriver } from 'wix-ui-test-utils/unidriver';
+import { dropdownDriverFactory as coreDriverFactory } from 'wix-ui-core/dist/src/components/dropdown/Dropdown.driver';
+
+import { DATA_HOOKS } from './constants';
+
+export interface FloatingDropdownDriver extends BaseUniDriver {
+  dropdownContentDisplayed(): Promise<boolean>;
+  clickOnDropdownBase(): Promise<void>;
+  selectOptionAt(i: number): Promise<void>;
+  getBaseSelectedValue(): Promise<string>;
+}
+
+export const floatingDropdownDriverFactory = (
+  base: UniDriver,
+): FloatingDropdownDriver => {
+  const baseUniDriver = baseUniDriverFactory(base);
+
+  const getDropdownBase = async () => {
+    return base.$$(selectorFromDataHook(DATA_HOOKS.base)).get(0);
```

Comment:
Just a remark:
I think you can use just `$` instead of `$$`, then you won't need the `get(0)`.
But it's really really ok like this as well

Reply:
Changed, thanks :)